### PR TITLE
improve devUX

### PR DESCRIPTION
**Pull request by amityadav0**

### CODE REVIEW

File: packages/algob/src/lib/status.ts [hunk 1]
```diff
@@ -28,7 +28,7 @@ export async function readGlobalStateSSC (
   appId: number): Promise<SSCStateSchema[] | undefined> {
   const accountInfoResponse = await deployer.algodClient.accountInformation(creator).do();
   for (const value of accountInfoResponse['created-apps']) {
```
Comment (robert-zaremba) [inline packages/algob/src/lib/status.ts:30]: please rename `value` to `app`

**Review by robert-zaremba — APPROVED** [review]
preapproving. left one change request.

File: packages/algorand-js/src/runtime/runtime.ts [hunk 2]
```diff
@@ -98,14 +98,24 @@ export class Runtime {
     return undefined;
   }
 
+  /**
+   * Fetches local state for account address and application index
+   * @param appId application index
+   * @param accountAddr address for which local state needs to be retrieved
+   */
+  getLocalState (appId: number, accountAddr: string, key: Uint8Array): StackElem | undefined {
```
Comment (ratik21) [inline packages/algorand-js/src/runtime/runtime.ts:106]: Please add one more param `key` in description

File: packages/algob/src/lib/status.ts (hunk 1 above)
Comment (amityadav0) [reply packages/algob/src/lib/status.ts:30]: ok

File: packages/algorand-js/src/runtime/runtime.ts (hunk 2 above)
Comment (amityadav0) [reply packages/algorand-js/src/runtime/runtime.ts:106]: ok

**Pull request merged**